tests/bin/migrate-post-content.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Migrates the entire site's post content to the new namespace.
+ *
+ * For testing, do not use in production.
```

Comment:
This WP-CLI script is mainly for our own testing. I don't think we should publicize this. 

It also checks that you really want to migrate:

<img width="544" alt="sure-migr" src="https://user-images.githubusercontent.com/4063887/84828910-70554300-afec-11ea-873e-ba8321d9d27a.png">